lit/docs/pipelines/managing.lit

```diff
@@ -138,3 +141,22 @@
   order. If there are other pipelines that you haven't included in the command,
   they may appear in-between, before, or after the given set.
 }
+
+\section{
+    \title{\code{fly archive-pipeline}}{fly-archive-pipeline}
+
+    Pipelines can be archived via fly. This means that the existing pipeline will
+    be paused and stays hidden on the concourse UI. Archived pipelines retains their
+    build logs associated with jobs, resources and resource types. When a new pipeline
```

Comment:
```suggestion
    build logs associated as well as jobs, resources and resource types. When a new pipeline
```
is this right? Or are you trying to say archived pipelines retain the logs from all three of these things? Because _build logs_ are by definition associated only to _jobs_.